ts-client/src/ValorClient.ts

```diff
@@ -0,0 +1,334 @@
+import axios, { AxiosInstance } from 'axios';
+
+type Dataset = {
+  id: number;
+  name: string;
+  metadata: object;
+};
+
+type Model = {
+  id: number;
+  name: string;
+  metadata: object;
+};
+
+type TaskType =
+  | 'skip'
+  | 'empty'
+  | 'classification'
+  | 'object-detection'
+  | 'semantic-segmentation'
+  | 'embedding';
+
+type Evaluation = {
+  id: number;
+  model_name: string;
+  datum_filter: object;
+  parameters: { task_type: TaskType; object };
+  status: 'pending' | 'running' | 'done' | 'failed' | 'deleting';
+  metrics: object[];
```

Comment:
are these not structured?

Reply:
they are a bit more structured-- just added more typing